cinema/serializers.py

```diff
@@ -1,6 +1,15 @@
 from rest_framework import serializers
 
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from django.db import transaction
+from cinema.models import (
+    Genre,
+    Actor,
+    CinemaHall,
+    Movie,
+    MovieSession,
+    Order,
+    Ticket
```

Comment:
Code Style issue: Group imports using `()` if needed. Your import statement is not formatted correctly. Here is the correct format:

python
from cinema.models import (
    Genre,
    Actor,
    CinemaHall,
    Movie,
    MovieSession,
    Order,
    Ticket,
)
```

Please, fix it according to the checklist.